go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/btcsuite/btcd v0.22.0-beta
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
 	github.com/cenkalti/backoff/v4 v4.0.2
+	github.com/go-jose/go-jose/v3 v3.0.1-0.20210613005559-6f2534b08aeb
```

Comment:
can't remove square/go-jose (line 33) in this PR?

Reply:
unfortunately not, due to modules depending on afgo revision. Following this pr, we should be able to remove square/go-jose completely.